dashboard/app/models/script_level.rb

```diff
@@ -133,6 +142,27 @@ def summarize
     summary
   end
 
+  # Given a script level summary for the last level in a stage, returns an
+  # array of additional levels if it is a multi-page assessment, or an empty
+  # array if not.
+  def self.summarize_extra_puzzle_pages(last_level_summary)
+    extra_levels = []
+    if last_level_summary[:kind] == "assessment"
+      level_info = Script.cache_find_level(last_level_summary[:id])
+      if level_info[:properties]["pages"] && level_info[:properties]["pages"].length > 1
```

Comment:
you can use the `long_assessment?` method that you just implemented above

I might have called level_info level instead, since it's an activerecord object (and not some other representation of a level)
also prefer `level.properties[]` instead of `level_info[:properties]`